ivy/functional/backends/paddle/elementwise.py

```diff
@@ -341,14 +342,7 @@ def less(
 
 
 @with_unsupported_dtypes(
-    {
-        "2.6.0 and below": (
-            "int8",
-            "int16",
-            "uint8",
-            "float16",
-        )
-    },
+    {"2.6.0 and below": ("int8", "uint8", "int16", "float16", "bfloat16")},
     backend_version,
```

Comment:
it's good and working fine. but i  think it's looks more understandable  if you add @supported_dtype(float32, float64, int32, int64 ,complex).

Reply:
Adding test support for `bool` also as it works in the main library.
```python
>>> x = paddle.to_tensor([[True, False], [False, True]])
>>> y = paddle.to_tensor([[False, False], [True, False]])
>>> x * y
Tensor(shape=[2, 2], dtype=bool, place=Place(cpu), stop_gradient=True,
       [[False, False],
        [False, False]])
```

Made the changes to handle `complex` dtype. Please review it.

Reply:
Hii @sgalpha01 the changes you made looking good to me. i have tested it's passing all the tests. 👍🏾